src/ui/public/index_patterns/_get.js

```diff
@@ -21,7 +21,12 @@ export function IndexPatternsGetProvider(Private) {
       fields: [],
```

Comment:
Should this be `fields: field ? [field] : []`?  cc @chrisronline.  If you pass `[]` I'm assuming it grabs all fields?  At least then there would be a reason to call getTitles/getIds because less data would be coming back from the server, since it'd only be requesting one field.  Though in the grand scheme of things, probably negligent performance improvement...